src/utils/material.js

```diff
@@ -118,11 +120,16 @@ function handleTextureEvents (el, texture) {
 
   // Video events.
   if (!texture.image || texture.image.tagName !== 'VIDEO') { return; }
+
   texture.image.addEventListener('loadeddata', function emitVideoTextureLoadedDataAll () {
     // Check to see if we need to use iOS 10 HLS shader.
     if (texture.needsCorrectionBGRA && texture.needsCorrectionFlipY) {
```

Comment:
One thing I was considering is passing `needsCorrectionBGRA` and `needsCorrectionFlipY` into shaders in general when true, so someone with a custom shader would know when corrections are necessary.  Thoughts?